README.md

```diff
@@ -67,9 +67,21 @@ to revise it in response.
 * Build consensus and integrate feedback. RFCs that have broad support
 are much more likely to make progress than those that don't receive any
 comments.
-* Eventually, somebody on the [core team] will either accept the RFC by
-merging the pull request and assigning the RFC a number, at which point
-the RFC is 'active', or reject it by closing the pull request.
+* Eventually, the [core team] will decide whether the RFC is a candidate
+for inclusion in Ember.
+* RFCs that are candidates for inclusion in Ember will enter a "final comment
+period", which shall be no shorter than 7 days. This period will be signaled
```

Comment:
It should be seven days; making it variable makes it harder for people to follow along.